test/service/test_response.py

```diff
@@ -1940,7 +1940,7 @@ def test_contributor_matrix_files(self):
             ('developmentStage', 'adult'): [
                 '4d6f6c96-2a83-43d8-8fe1-0f53bffd4674.HumanLiver.zip',
             ],
-            ('organ', 'liver'): [
+            ('specimenOrgan', 'liver'): [
```

Comment:
And what's this? This seems unrelated. Or is this the real fix and the enumeration above is cosmetic?